app/views/ladder/SimulateTournamentTabView.coffee

```diff
@@ -0,0 +1,164 @@
+CocoView = require 'views/core/CocoView'
```

Comment:
sounds reasonable. let me make all simulators fetch the normal ladder and tournament `next-match` at the same time

Reply:
Server could decide instead of client making multiple requests. https://github.com/codecombat/codecombat-server/blob/master/server/middleware/levels.coffee#L387-L390 already decides whether to pull a queued match from Redis `ladder-match-queue` or query for one randomly from `ladder-rankings`, could always pull from the tournament match queue as first priority.